src/main/resources/data-h2.sql

```diff
@@ -0,0 +1,2 @@
nse-reference", "LicenseRef-scancode-unknown-license-reference", "UNREVIEWED"), (618, NULL, "LicenseRef-scancode-other-permissive", "LicenseRef-scancode-other-permissive", "UNREVIEWED"), (623, NULL, "GPL-2.0 WITH Linux-syscall-note", "GPL-2.0 WITH Linux-syscall-note", "UNREVIEWED"), (624, NULL, "LicenseRef-scancode-unicode", "LicenseRef-scancode-unicode", "UNREVIEWED"), (625, NULL, "Classpath-exception-2.0", "Classpath-exception-2.0", "UNREVIEWED"), (626, NULL, "LicenseRef-scancode-oracle-openjdk-classpath-exception-2.0", "LicenseRef-scancode-oracle-openjdk-classpath-exception-2.0", "UNREVIEWED"), (627, NULL, "LicenseRef-scancode-proprietary-license", "LicenseRef-scancode-proprietary-license", "UNREVIEWED"), (628, NULL, "LicenseRef-scanoss-pop-tech-pty-ltd-sdk-license;-version-1.0-4806", "LicenseRef-scanoss-pop-tech-pty-ltd-sdk-license;-version-1.0-4806", "UNREVIEWED"), (629, NULL, "LicenseRef-scanoss-pop-tech-pty-ltd-sdk-license;-version-1.0-6787", "LicenseRef-scanoss-pop-tech-pty-ltd-sdk-license;-version-1.0-6787", "UNREVIEWED"), (630, NULL, "LicenseRef-scancode-commercial-license", "LicenseRef-scancode-commercial-license", "UNREVIEWED"), (631, NULL, "LicenseRef-scancode-generic-cla", "LicenseRef-scancode-generic-cla", "UNREVIEWED"), (632, NULL, "LicenseRef-scancode-warranty-disclaimer", "LicenseRef-scancode-warranty-disclaimer", "UNREVIEWED"), (633, NULL, "LicenseRef-scancode-unknown", "LicenseRef-scancode-unknown", "UNREVIEWED"), (634, NULL, "LLVM-exception", "LLVM-exception", "UNREVIEWED"), (635, NULL, "LicenseRef-scanoss-flexmonster-terms-and-conditions-1679", "LicenseRef-scanoss-flexmonster-terms-and-conditions-1679", "UNREVIEWED"), (636, NULL, "LicenseRef-scanoss-flexmonster-terms-and-conditions-2985", "LicenseRef-scanoss-flexmonster-terms-and-conditions-2985", "UNREVIEWED"), (637, NULL, "LicenseRef-scancode-free-unknown", "LicenseRef-scancode-free-unknown", "UNREVIEWED"), (638, NULL, "LicenseRef-scanoss-commercial-3870", "LicenseRef-scanoss-commercial-3870", "UNREVIEWED"), (639, NULL, "LicenseRef-scanoss-commercial-2133", "LicenseRef-scanoss-commercial-2133", "UNREVIEWED"), (640, NULL, "Linux-syscall-note", "Linux-syscall-note", "UNREVIEWED"), (641, NULL, "LicenseRef-scancode-other-copyleft", "LicenseRef-scancode-other-copyleft", "UNREVIEWED"), (642, NULL, "LicenseRef-scancode-public-domain", "LicenseRef-scancode-public-domain", "UNREVIEWED"), (643, NULL, "GPL-2.0-only-WITH-Linux-syscall-note", "GPL-2.0-only-WITH-Linux-syscall-note", "UNREVIEWED"), (644, NULL, "LicenseRef-scancode-facebook-patent-rights-2", "LicenseRef-scancode-facebook-patent-rights-2", "UNREVIEWED"), (645, NULL, "NAIST-2003", "NAIST-2003", "UNREVIEWED"), (646, NULL, "LicenseRef-scancode-philippe-de-muyter", "LicenseRef-scancode-philippe-de-muyter", "UNREVIEWED"), (647, NULL, "LicenseRef-scanoss-proprietary-4209", "LicenseRef-scanoss-proprietary-4209", "UNREVIEWED"), (648, NULL, "LicenseRef-scanoss-proprietary-6843", "LicenseRef-scanoss-proprietary-6843", "UNREVIEWED");
```

Comment:
What about building the license database with [orori api](https://osori-db.github.io/en/docs/guide/api-doc/) instead of saving this manually?